models/apple_search_ads.yml

```diff
@@ -235,7 +235,12 @@ models:
       - dbt_utils.unique_combination_of_columns:
           combination_of_columns:
             - search_term_text
+            - match_type
             - date_day
+            - keyword_id
+            - ad_group_id
+            - campaign_id
+            - organization_id
```

Comment:
I noticed you added the `organization_id` and `match_type` fields to the uniqueness test, which the customer didn't have in their initial PR. My instinct would be to include only the fields we are marking as keys from the ERD of Apple Search Ads. Was there additional information that informed you to add the above two fields?

![image](https://github.com/fivetran/dbt_apple_search_ads/assets/74217849/8c0fd500-3171-4656-baf5-0f2ab710e43f)

Reply:
If the above is updated we should also remember to update the CHANGELOG.

Reply:
Yes `organization_id` and `match_type` were added to keep standard to what we use to test in the [ad reporting package](https://github.com/fivetran/dbt_ad_reporting/blob/cb0738c6c5b9b7bf2c8598638816d3401ae06fc0/models/ad_reporting_models.yml#L183-L195). I think it makes sense to go based off what's in the ERD (pending us being notified what fivetran_id is hashed from) though. 

But do you think there's any downside to adding those fields as well?

Reply:
I don't foresee there being any downside with including them. Especially if we already test on these fields uniqueness in the downstream ad reporting model. I am comfortable keeping the test with how you have it in this PR.